libobs/util/platform.h

```diff
@@ -201,6 +201,10 @@ EXPORT bool os_get_proc_memory_usage(os_proc_memory_usage_t *usage);
 EXPORT uint64_t os_get_proc_resident_size(void);
 EXPORT uint64_t os_get_proc_virtual_size(void);
 
+#define UUID_STR_LENGTH 36
+
+EXPORT char *os_get_uuid(void);
```

Comment:
Perhaps `os_generate_uuid` would be a clearer name for this function, `get` could imply it returns something that already exists.

Reply:
Makes sense. I'll update the name.